lib/UserConfig/UserConfig.js

```diff
@@ -0,0 +1,30 @@
+import { join } from "node:path";
+
+class UserConfig {
+  constructor(objArgs) {
+    this.fileName =
+      (objArgs?.fileName && `${objArgs.fileName}.config.js`) ||
+      "powerPlop.config.js";
+    this.dirPath =
+      (objArgs?.dirPath && join(process.cwd(), objArgs.dirPath)) ||
+      process.cwd();
+    this.fullPath = join(this.dirPath, this.fileName);
+  }
+
+  async root() {
+    const config = await this.retrieveConfig()
+    return config.root;
+  }
+
+  async initType() {
+    const config = await this.retrieveConfig()
+    return config.initType;
+  }
+
+  async retrieveConfig() {
+    const config = await import(this.fullPath);
+    return config.default;
+  }
```

Comment:
Async seemd to be the only way to dynamically import the config file. Could have used a JSON and used fs.readFileSync(). JSON is static and prefer to have config file be js.